packages/components/collapsible-panel/src/collapsible-panel.tsx

```diff
@@ -20,9 +21,11 @@ import {
   HeaderControlsWrapper,
   SectionWrapper,
   SectionContent,
+  getBaseContainerStyles,
+  SectionDescriptionWrapper,
 } from './collapsible-panel.styles';
 import CollapsiblePanelHeader from './collapsible-panel-header';
-
+import { ThemeName, useTheme } from '@commercetools-uikit/design-system';
```

Comment:
```suggestion
import { type ThemeName, useTheme } from '@commercetools-uikit/design-system';
```

Reply:
Updated in this commit: b2c37088f4d207992c3a579236e88828daa233b6